api_tests/lib/actor.ts

```diff
@@ -153,8 +153,13 @@ export class Actor {
 
             if (
                 field.class.some((e: string) => e === "bitcoin") &&
-                field.class.some((e: string) => e === "feePerByte")
+                field.class.some((e: string) => e === "feePerWU")
             ) {
+                expect(field.class).contains(
```

Comment:
Would this break comit-i? if so, let's wait Wednesday to merge

Reply:
> Do we want to start with backwards compatibility support already?

Not supporting this will break bobtimus, hence I'd keep it and remove as soon as bobtimus is updated.

Reply:
Can we have an issue that tracks on Bobtimus side please?

Reply:
@thomaseizinger please create an issue on bobtimus side

Reply:
Done: https://github.com/coblox/bobtimus/issues/42